main.py

```diff
@@ -0,0 +1,42 @@
+import logging
+import os
+import sys
+
+import yaml
+
+sys.path.append(os.path.join(os.path.dirname(__file__), "src"))
```

Comment:
This works. What I also recommend to try is to package your custom code, create .whl file and install using pip to make it available in your environment.

Reply:
Thanks Basak, it was in the video indeed, changed it now and it ran